yarn.lock

```diff
@@ -5941,13 +5941,20 @@ __metadata:
   languageName: node
   linkType: hard
 
-"comlink@npm:^4.3.1, comlink@npm:^4.4.1":
+"comlink@npm:^4.3.1":
   version: 4.4.1
   resolution: "comlink@npm:4.4.1"
   checksum: 16d58a8f590087fc45432e31d6c138308dfd4b75b89aec0b7f7bb97ad33d810381bd2b1e608a1fb2cf05979af9cbfcdcaf1715996d5fcf77aeb013b6da3260af
   languageName: node
   linkType: hard
 
+"comlink@npm:^4.4.2":
```

Comment:
Looks like the two versions of `comlink` could be deduplicated with:

https://github.com/jupyterlite/pyodide-kernel/blob/fb9c10dc5d149849d04dffbc073680c64c988993/package.json#L20

Reply:
Ok, I will try this. I am not so fluent with yarn, I normally use npm. I'll be back in the evening at my dev computer.

Reply:
Running `jlpm deduplicate` at the top-level should normally perform the deduplication step.